src/types.js

```diff
@@ -10,24 +10,44 @@ import type {FusionPlugin, Token} from 'fusion-core';
 
 type FontURLsType = {
   woff?: string,
-  woff2?: string,
+  woff2: string,
 };
+
 type FontFallbackType = {
   name: string,
   styles?: {
     [string]: string,
   },
 };
-export type FontType = {
+
+export type AtomicFontType = {
   urls: FontURLsType,
-  fallback: FontFallbackType,
+  fallback?: FontFallbackType,
+  styles?: {},
+};
+
+export type AtomicFontsObjectType = {
+  [string]: AtomicFontType,
+};
+
+export type StyledFontsObjectType = {
+  [string]: Array<AtomicFontType>,
 };
+
 export type ConfigType = {
```

Comment:
Ah, nevermind.  Here it is!  Disregard earlier comment, assuming we export this in `index.js` as well.